cluster/manifests/01-admission-control/config.yaml

```diff
@@ -35,6 +35,7 @@ data:
   pod.env-inject.variable._PLATFORM_OBSERVABILITY_METRICS_PORT: "{{ .Cluster.ConfigItems.observability_metrics_port }}"
   pod.env-inject.variable._PLATFORM_OBSERVABILITY_ACCESS_TOKEN: "{{ .Cluster.ConfigItems.lightstep_token }}"
   pod.env-inject.variable._PLATFORM_OBSERVABILITY_COMMON_ATTRIBUTE_CLOUD__ACCOUNT__ID : "{{ .Cluster.Alias }}"
+  pod.env-inject.variable._STACKSET_HPA_FIELD_ENABLED: "{{ .Cluster.ConfigItems.stackset_hpa_field_enabled }}"
```

Comment:
These are the environment variables that will be injected in pods. It should not be here, but rather a new config option similiar to e.g. `pod.parent-resource-hash.enable` above.

Reply:
@mikkeloscar Is `stackset.default.enable_hpa_field` ok? Or like `pod.stackset-def.enable_hpa_field`?

Reply:
Maybe `stackset.legacy_hpa_field.enable` to follow the naming off the other fields.